_data/simulations/CHiMaD3_MOOSE/meta.yaml

```diff
@@ -0,0 +1,116 @@
+_id: 5060d3a0-7b16-11e8-b5f4-6170e8cb26a5
+metadata:
+  author:
+    first: Dong-Uk
+    last: Kim
+    email: cmskdu@gmail.com
+    github_id: eastglow-zz
+  timestamp: '27 June, 2018'
+  summary: >-
+    FiPy implementation of benchmark 3a on a 960x960 grid. The shape of the
```

Comment:
This claims that you are using FiPy, but you're also claim to use Moose. Can you edit this?